tasks/main.yml

```diff
@@ -11,6 +11,12 @@
 - name: Installing task {{ name }}
   template: src=task.conf.j2 dest={{ supervisor_config_dir }}/{{ name }}.conf
   sudo: yes
+  when: state == "present"
```

Comment:
I think these should be wrapped in curly braces to make Ansible 2 happy `"{{ state == 'present' }}"`